python/tvm/relay/frontend/pytorch.py

```diff
@@ -3919,7 +3950,8 @@ def _handel_nested_input(inputs):
                     out_names = _get_output_names(op_node)
                     outputs.update(zip(out_names, relay_out))
                 else:
-                    assert op_node.outputsSize() == 1
+                    # The node_name of embedding_bag is like "22_23_24_25"
```

Comment:
this change looks very clean to me. We may want to check if this node is `embedding_bag` before we do the split. My concern is that this change may not work on other ops that return more than one outputs.

Reply:
yeah but my point is that 

1) Other ops which have one output will enter this branch. And this line essentially does nothing for those ops, which could confuse others who don't have a better context of the reason we add this split here.
2) if there is other future ops that also have more than one outputs but got concatenated into one, like previously what we did with `embedding_bag`, this behavior may not be ideal/expected. 

Hence my take on this is that we add a check like
```
if this is embedding_bag:
    # comment that specify the reason why we only do split for embedding_bag 
    we split the output by `-`
```
which could make the codebase much cleaner.